src/scss/style.scss

```diff
@@ -3,4 +3,9 @@
 .error {
   @extend .text-primary-red;
   @extend .border-primary-red;
+}
+
+.info {
```

Comment:
@devowhippit these colors correspond to the `bg-green-100` and `border-green-700` tailwind classes, but it didn't look like I was able to use these classes in the screening docs? Are they not imported by the nyco patterns? 

Is what I've done here ok, or would it be better to add these tailwind colors to nyco patterns and then import them into screening docs?